functions/helpFunctions.tsx

```diff
@@ -47,25 +47,46 @@ export function insperaDataToTextboxObject(
 }
 
 export function saveAssessments(
-  assessments: AssessmentType[],
-  key: number
+  assessments: ApprovalType[],
+  key: string
 ): void {
   // local storage is a property of object window. Accessing localStroage is not possible until React component has mounted
-  if (typeof window !== "undefined") {
+  if (typeof window !== 'undefined') {
     // store array as a string
-    localStorage.setItem(key.toString(), JSON.stringify(assessments));
+    localStorage.setItem(key, JSON.stringify(assessments));
+  }
+}
+
+export function getAssessments(taskNumbers: number[]): number[][] {
+  //convert tasknumbers to string
+  const tasks = taskNumbers.map(String);
```

Comment:
Why `String` in map? Is it stringifying the list of numbers?

Reply:
Correct ^💯

Reply:
Damn, cool:)